examples/wasm/index.html

```diff
@@ -0,0 +1,4 @@
+<script type="module">
```

Comment:
can we wrap this in the standard html boilerplate to suppress browser errors/warnings?

```html
<html>
  <head>
    <meta charset="UTF-8">
  </head>
  <script type="module">
    import init from './target/headless_wasm.js'
    init()
  </script>
</html>
```

Reply:
This is perfectly valid HTML.
Your browser is overzealous. ;-)

But I do see your point.